tsconfig-type-test.json

```
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "target": "es2018",
```

```suggestion
    "target": "es2022",
```

We support Node.js 18 or newer so we can target ES 2022

https://github.com/markedjs/marked/blob/a26f002d1819462c6cae75bb0c26dc064e17b4db/package.json#L91-L93